app/views/content_items/consultation.html.erb

```diff
@@ -1,175 +1,193 @@
-<%= render 'shared/history_notice', content_item: @content_item %>
-
-<% if @content_item.unopened? %>
-  <% content_item_unopened = capture do %>
-    This consultation opens
-    <% if @content_item.opening_date_midnight? %>on<% else %>at<% end %>
-    <time datetime="<%= @content_item.opening_date_time %>"><%= @content_item.opening_date %></time>
-  <% end %>
-  <%= render 'components/notice', title: "This consultation isn't open yet", description_text: content_item_unopened %>
-<% elsif @content_item.pending_final_outcome? %>
-  <% content_item_final_outcome = capture do %>
-    Visit this page again soon to download the outcome to this public&nbsp;feedback.
-  <% end %>
-  <%= render 'components/notice', title: 'We are analysing your feedback', description_text: content_item_final_outcome %>
-<% elsif @content_item.final_outcome? %>
-  <%= render 'components/notice', title: 'This consultation has concluded' %>
-
-  <% if @content_item.final_outcome_documents? %>
-    <div class="grid-row sidebar-with-body">
-      <div class="column-third">
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/title', @content_item.title_and_context %>
+  </div>
+  <%= render 'shared/translations' %>
+  <div class="column-two-thirds">
+    <%= render 'components/notice', @content_item.withdrawal_notice_component  %>
+    <%= render 'shared/history_notice', content_item: @content_item %>
+  </div>
+</div>
+
+<%= render 'shared/publisher_metadata_with_logo', content_item: @content_item %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'components/important-metadata', items: @content_item.metadata[:other]  %>
+
+    <% if @content_item.unopened? %>
+      <% content_item_unopened = capture do %>
+        This consultation opens
+        <% if @content_item.opening_date_midnight? %>on<% else %>at<% end %>
+        <time datetime="<%= @content_item.opening_date_time %>"><%= @content_item.opening_date %></time>
+      <% end %>
+      <%= render 'components/notice', title: "This consultation isn't open yet", description_text: content_item_unopened %>
+
+    <% elsif @content_item.pending_final_outcome? %>
+      <% content_item_final_outcome = capture do %>
+        Visit this page again soon to download the outcome to this public&nbsp;feedback.
+      <% end %>
+      <%= render 'components/notice', title: 'We are analysing your feedback', description_text: content_item_final_outcome, margin_bottom: 0 %>
+
+    <% elsif @content_item.final_outcome? %>
+      <%= render 'components/notice', title: 'This consultation has concluded' %>
+
+      <% if @content_item.final_outcome_documents? %>
         <%= render 'components/heading', text: "Download the full outcome", id: "final-outcome-documents-title" %>
-      </div>
-      <div class="column-two-thirds" aria-labelledby="final-outcome-documents-title">
+        <div aria-labelledby="final-outcome-documents-title">
```

Comment:
As the heading and content are now inline, we could probably remove this div and the aria attribute.